src/Example.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import { determineWeather } from "./util/determineWeather";
+import { Card } from "./components/Card";
+import { Temperature } from "./components/Temperature";
+
+export const Example = () => {
+  const [card1Temperature, setCard1Temperature] = useState(110);
+  const [card2Temperature, setCard2Temperature] = useState(65);
+  const [card3Temperature, setCard3Temperature] = useState(30);
+
+  const DELTA = 5;
+
+  const increaseCard1Temperature = () =>
+    setCard1Temperature(card1Temperature + DELTA);
+  const decreaseCard1Temperature = () =>
+    setCard1Temperature(card1Temperature - DELTA);
+
+  const increaseCard2Temperature = () =>
+    setCard2Temperature(card2Temperature + DELTA);
+  const decreaseCard2Temperature = () =>
+    setCard2Temperature(card2Temperature - DELTA);
+
+  const increaseCard3Temperature = () =>
+    setCard3Temperature(card3Temperature + DELTA);
+  const decreaseCard3Temperature = () =>
+    setCard3Temperature(card3Temperature - DELTA);
+
+  return (
+    <div className="card-container">
+      <Card width={150} height={150} type={determineWeather(card1Temperature)}>
```

Comment:
Components always have named arguments. I have width, height, and type for card.